student-source/courses.rst

```diff
@@ -51,7 +51,7 @@ The student courses are displayed in the right pane, as shown below with the def
     - Click the arrow to expand or collapse the panel. 
     - Click the feedback panel to open it and review your assessments, grading comments, and any code comments from your instructor.
 
-**4** - **Courses** panel is the main panel on the page and is where you can see the courses, modules, and assignments that have been assigned to you, and the end date for the assignment. Up to five pinned assignments can be displayed more prominently at the top of the page. From this area, you can easily re-open the assignment you were last working on, or start any of the other assignments that have been assigned to you. You start or open an assignment using the navigation buttons.
+**4** - **Courses** panel is the main panel on the page and is where you can see the courses, modules, if enabled by your teacher/instructor a progress bar showing your overall progress and assignments that have been assigned to you, and the end date for the assignment. Up to five pinned assignments can be displayed more prominently at the top of the page. From this area, you can easily re-open the assignment you were last working on, or start any of the other assignments that have been assigned to you. You start or open an assignment using the navigation buttons.
```

Comment:
**Courses** panel is the main panel on the page and is where you can see the courses, modules and assignments that have been assigned to you, and the end date for the assignment. If your teacher/instructor has enabled the feature, you will also see a progress bar displaying your overall progress.